AnnoyancesFilter/sections/tweaks.txt

```diff
@@ -1438,6 +1438,7 @@ lezhinus.com,lezhin.com###viewer-coach
 qanda.ai##div[class^="Picture__Protector-"]
 codedosa.com###wpcp-error-message
 !--- CSS ---
+toptoon.com#$#.document_img { pointer-events: auto!important; }
```

Comment:
Please make an extra space after the property value in CSS rules.
```
toptoon.com#$#.document_img { pointer-events: auto !important; }
```

Reply:
Done. https://github.com/AdguardTeam/AdguardFilters/pull/98854/commits/05c8922fe869c4831b5026f5d456983e3618fa3e